### DISCUSSION TITLE
Waiting for image transmission - never gets connect

### DISCUSSION THREAD

**Discussion by tootai** [rustdesk/rustdesk#8258] [General]
Hi
I setup my own server and use a Linux 1.2.3 version to connect to customers running a 1.2.3-2 version. Connection is OK, get "Success. Connected waiting for image transmission" (translated from french message) and after some time I get disconnected, immediately a new connection is started by the software, then again get the success message and all start again like explained above.

What's wrong here ?

Daniel

**Comment by hassansf** [comment]
Same issue here. I've downgraded the server to older version even the client app to older and newer versions but no avail. The machine I am trying to access never displays but apparently if I reverse the connection, it works. That leaves me with two possible culprits. Either it is the Firewall or the machine itself. Still working on it....

**Comment by 21pages** [comment]
Could you try https://github.com/rustdesk/rustdesk/discussions/9124#discussioncomment-10391439?

**Comment by rustdesk** [comment]
Close and lock to favor https://github.com/rustdesk/rustdesk/discussions/9124#discussioncomment-10391439
